docs/reference/converter-lang.adoc

```diff
@@ -72,6 +72,25 @@ closure::
 Converts the listed nodes, all descendants, and all referenced nodes.
 We apply this strategy until we include all Annotation, Containment, and Reference targets. 
 
+
+.Inclusion of nodes and ids
```

Comment:
I still find this hard to grok. A diagram might help, but I think that I should make an effort myself, then :)

Reply:
Removed the node, as it's not relevant in default settings.

I'd still happily receive a better explanation (-: